src/data/faq.json

```diff
@@ -28,7 +28,19 @@
                             "As a media representative, please direct your enquiries to <a href='mailto:press@sap.com'>press@sap.com</a>.",
                             "The various feedback channels of the <a href='https://www.coronawarn.app/de/community/'>Corona-Warn-App-Community<a/> will help you with your questions about the open source project of the Corona-Warn-App."
                         ]
+                    },
+                    {
+                        "title": "How to reach us from abroad - International phone numbers",
+                        "anchor": "international_phone_numbers",
+                        "active": true,
+                        "textblock": [
+                            "From abroad you can reach us at the following international phone numbers:",
+                            "Technical hotline: +49 30 498 75401 available Monday to Saturday from 7 a.m. to 10 p.m. (except on national holidays)",

```

Comment:
> 7 a.m. to 10 p.m.

Should mention that this is local time in the GMT+1 timezone?

Reply:
ok, I guess most user will not understand GMT+1 timezone.

Maybe:
German local time?

Reply:
yeah that should be okay

Reply:
ok